lib/public/views/Runs/formatRunDuration.js

```diff
@@ -50,3 +52,29 @@ export const formatRunDuration = (run, raw = false) => {
         return h(`strong.${ret.classes}`, ret.text);
     }
 };
+
+/**
+ * Compares the given values and checks how many asteriks needs to be given to the duration.
+ * @param {Run} run the current run object.
+ * @returns {String} one/two asteriks or an empty string
+ */
+export const _getAsteriks = (run) => {
```

Comment:
I do not think the _ is useful, neither is the export for now

Reply:
Fixed in next commit